lib/internal/Magento/Framework/Reflection/Test/Unit/_files/TSample.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Reflection\Test\Unit;
+
+class TSample implements TSampleInterface
+{
+    /**
+     * {@inheritdoc}
```

Comment:
Such syntax only inheritance method description, not type hints for parameters and return value.

See https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#61-making-inheritance-explicit-using-the-inheritdoc-tag

Please change implementation in such a way that we do not allow misuse of syntax with curly braces and only inherit return/param in case of `@inheritdoc`.

Reply:
According to Magento internal guidelines, this annotation should look like `{@inheritdoc}`.

Reply:
Where do you see it? http://devdocs.magento.com/guides/v2.2/coding-standards/docblock-standard-general.html#inheritdoc is compliant with what I'm saying.

Reply:
From internal Magento wiki but I agree with `@inheritdoc` usage.
![screenshot from 2017-10-19 15-38-51](https://user-images.githubusercontent.com/2736528/31771175-cbde7478-b4e3-11e7-8b0a-65d7dd563f22.png)

Reply:
Great 👍 Yeah, seems to be some outdated documentation, please change/remove it or delegate this to whoever is in charge of it.